moduleroot/Rakefile

```diff
@@ -6,4 +6,4 @@ PuppetLint.configuration.fail_on_warnings = true
 <% checks.each do |check| -%>
 PuppetLint.configuration.send('<%= check %>')
 <% end -%>
-PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
```

Comment:
This line caused me a lot of issues in the Windows module space.  The puppetlabs-spec-helper already sets a default ignore which is for more broad

`config.ignore_paths = ["tests/**/*.pp", "vendor/**/*.pp","examples/**/*.pp", "spec/**/*.pp", "pkg/**/*.pp"]`

https://github.com/puppetlabs/puppetlabs_spec_helper/blob/master/lib/puppetlabs_spec_helper/rake_tasks.rb#L392